### PR TITLE
bug 1603122 clarifying need for etcd v2 and v3 datastores

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1603122

@wanghaoran1988, @sdodson, will you PTAL?

### CODE REVIEW

**Comment by xingxingxia** [comment]
Hi, about QE side, Haoran transferred to SRE (OPS) team, I transferred to be his role.
I reviewed the PR and didn't find issue.
Ge Liu is more focused on etcd. @geliu2016  please help review for double confirm, thx

**Comment by sdodson** [comment]
I've commented on the bug as well, but my opinion is that the docs are fine as is without this change. If there's ever a scenario where we can prevent the admin from having to make a choice we should simply remove that choice and I think that's what the documentation does currently by instructing them to backup both v2 and v3 data stores.

**Comment by kalexand-rh** [comment]
Thank you @xingxingxia!

@sdodson, I made another update based on your comments in the bug. Will you PTAL?

File: day_two_guide/topics/proc_backing-up-etcd.adoc [hunk 1]
```diff
@@ -201,7 +201,13 @@ Back up the etcd data:
     --backup-dir /backup/etcd-$(date +%Y%m%d)
 # cp /var/lib/etcd/member/snap/db /backup/etcd-$(date +%Y%m%d)
 ----
-* If you use the v3 API, run the following command:
+* If you use the v3 API, run the following commands:
```
Comment (sdodson) [inline day_two_guide/topics/proc_backing-up-etcd.adoc]: The steps below on lines 213-220 are complete for both v2 and v3, lets get rid of lines 189-204.

Comment (geliu2016) [reply day_two_guide/topics/proc_backing-up-etcd.adoc]: @sdodson @kalexand-rh , sorry for confusing about this comment, 213-220 is command lines for v3, and 189-204 is for v2, the command lines are similar but not same completely, at least the command lines' head is etcdctl2 VS. etctctl3, as my personal understanding, current description is more clear for customer to follow. do you think so? thanks!

Comment (geliu2016) [reply day_two_guide/topics/proc_backing-up-etcd.adoc]: @sdodson @kalexand-rh , just curious, as we know, in current ocp env, we run command 'etcdctl' is etcdctl2, and 'ETCDCTL_API=3 etcdctl' is etcdctl3, so why the doc still using: etcdctl2 & etcdctl3? actually, ectdctl2 & etcdctl3 is invalid command in opc env just like this:

sh-4.2# etcdctl2
sh: etcdctl2: command not found
sh-4.2# 
sh-4.2# etcdctl3
sh: etcdctl3: command not found
sh-4.2# 
sh-4.2# 
sh-4.2# 
sh-4.2# etcdctl --cert-file=/etc/etcd/peer.crt  --key-file=/etc/etcd/peer.key --ca-file=/etc/etcd/ca.crt --peers=https://qe-geliu3xxx-etcd-1:2379 member list
72faa1d29ffc61d4: name=qe-geliu3master-etcd-1 peerURLs=https://10.xxx.0.xx:2380 clientURLs=https://10.xxx.0.xx:2379 isLeader=true
sh-4.2# 
sh-4.2# 
sh-4.2# ETCDCTL_API=3 etcdctl --cert=/etc/etcd/peer.crt  --key=/etc/etcd/peer.key --cacert=/etc/etcd/ca.crt --endpoints=https://qe-geliu3xxx-etcd-1:2379 endpoint health
https://qe-geliu3xxx-etcd-1:2379 is healthy: successfully committed proposal: took = 1.153385ms

Comment (kalexand-rh) [reply day_two_guide/topics/proc_backing-up-etcd.adoc]: @geliu2016, if the commands aren't exactly the same, I think we should keep them separate. (Unless no one should be using the etcd2 commands anymore. Then I think we should remove them.)
 I don't know! I can open a new PR to replace "etcdctl2" with "etcdctl" and replace "etcdctl3" with "ETCDCTL_API=3" for us to discuss. Would you like to open a new bug for this issue or try to handle it on this bug?

Comment (geliu2016) [reply day_two_guide/topics/proc_backing-up-etcd.adoc]: @kalexand-rh , I think we reach the agreement for first issue, and I will close the on_qa bug, regarding to 2nd issue, I file a new bug to trace it, https://bugzilla.redhat.com/show_bug.cgi?id=1610166, thx

Comment (sdodson) [reply day_two_guide/topics/proc_backing-up-etcd.adoc]: `etcdctl2` and `etcdctl3` are bash functions that should be deployed to /etc/profile.d/etcdctl.sh

line 214 backs up v3 data, lines 217-219 are commands that backup v2 data.

**Comment by kalexand-rh** [comment]
Based on the discussion on this bug, we're going to keep the docs the way they are. It's better to run an extra command than risk missing the other data store.

File: day_two_guide/topics/proc_backing-up-etcd.adoc (hunk 1 above)
Comment (kalexand-rh) [reply day_two_guide/topics/proc_backing-up-etcd.adoc]: @sdodson, can we finish the discussion about `etcdctl2` and `etcdctl3` on https://bugzilla.redhat.com/show_bug.cgi?id=1610166?

**Comment by kalexand-rh** [comment]
I think it's fair to add the note stating that you're running the commands to back up both data stores, though.

@openshift/team-documentation, PTAL

**Comment by kalexand-rh** [comment]
Thanks @mburke5678!

**Pull request merged**